inf-clojure.el

```diff
@@ -688,8 +708,12 @@ If `comint-use-prompt-regexp' is nil (the default), \\[comint-insert-input] on
   (setq comint-input-sender 'inf-clojure--send-string)
   (setq comint-prompt-regexp inf-clojure-comint-prompt-regexp)
   (setq mode-line-process '(":%s"))
-  (clojure-mode-variables)
-  (clojure-font-lock-setup)
+  ;; We don't need to call it in Clojure buffers.
+  (unless (inf-clojure--clojure-buffer-p)
```

Comment:
I don't get the need for this check - after all that's the init of the REPL's major mode.

Reply:
Indeed, my bad. For some reason I thought it's used for Clojure source buffers as well. Will revert it.

Reply:
Reverted. Thanks!